python/functional-programming/functional-programming-ii/README.md

```diff
@@ -0,0 +1,9 @@
+name: Functional Programming II
+
+description: Programming like mathematicians.

```

Comment:
we should think of a better description as this one is already used in Functional Programming

Reply:
I've added a new description